package.json

```diff
@@ -69,6 +69,6 @@
     "typescript": "^5.1.3"
   },
   "peerDependencies": {
-    "aws-cdk-lib": "^2.67.0"
+    "aws-cdk-lib": "^2.87.0"
```

Comment:
`LambdaInsightsVersion.VERSION_1_0_229_0` was introduced in [v2.87.0](https://github.com/aws/aws-cdk/releases/tag/v2.87.0).